### DISCUSSION TITLE
Any way to prevent error-squigglies on the indent guides?

### DISCUSSION THREAD

**Discussion by Hubro** [lukas-reineke/indent-blankline.nvim#569] [Q&A] [answered]
When LSP servers produce errors for blocks of code like this:

![image](https://user-images.githubusercontent.com/597206/235407957-3d898f49-a68d-483a-adcd-47576a8549f8.png)

The indent guides also get the error underline, which looks super messy and buggy.

Is there any way to prevent this?

**Comment by lukas-reineke** [comment]
This looks like a problem with your color scheme.
Can you post the output of `:highlight IndentBlanklineChar`?

**Reply by Hubro** [reply]
```
:highlight IndentBlanklineChar
IndentBlanklineChar xxx guifg=#313244
```

**Reply by lukas-reineke** [reply, marked as answer]
Yeah this is the problem. It is also mentioned in the documentation.

> When defining the highlight group, it is important to set `nocombine` as a
gui option. This is to make sure the character does not inherit gui options
from the underlying text, like italic or bold.

You need `gui=nocombine` on that highlight group.
If this is your own color scheme, just add it. If not, make a PR or issue in that repo to fix it.
